sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -44,7 +44,21 @@ const Lottery = (props: { listing: Listing }) => {
   const [loading, setLoading] = useState(false)
 
   const { listingsService, lotteryService, profile } = useContext(AuthContext)
-  const { onExport, exportLoading } = useLotteryExport(listing?.id)
+
+  const listingJurisdiction = profile?.jurisdictions?.find(
+    (jurisdiction) => jurisdiction.id === listing?.jurisdictions.id
+  )
+
+  const includeDemographicsPartner =
+    profile?.userRoles?.isPartner && listingJurisdiction?.enablePartnerDemographics
+
+  const { onExport, exportLoading } = useLotteryExport(
```

Comment:
Couldn't we do this all on the backend by looking at the requesting user?

Reply:
This is where we call the backend. Are you referring to the includeDemographics logic? The include demographics query param follows the same pattern as the csv export. We could move it to the backend but then we would also need to look up the jurisdiction to know if partnerdemographics is enabled

Reply:
Yeah we don't have to make this change here, but I think it makes more sense to do all of that on the backend